cron/centAcl-Func.php

```diff
@@ -418,55 +418,55 @@ function hasServiceCategoryFilter($res_id)
     return false;
 }
 
-function getAuthorizedServicesHost($host_id, $res_id, $authorizedCategories)
+function getAuthorizedServicesHost($hostId, $resId, $authorizedCategories)
 {
-    global $pearDB, $svcCache, $hostCache;
+    global $pearDB;
 
-    $tab_svc = getMyHostServicesByName($host_id);
+    $tabSvc = getMyHostServicesByName($hostId);
 
     /*
      * Get Service Groups
      */
-    $svc_SG = getACLSGForHost($pearDB, $host_id, $res_id);
+    $svcSg = getACLSGForHost($pearDB, $hostId, $resId);
 
-    $tab_services = array();
+    $tabServices = array();
     if (count($authorizedCategories)) {
-        if ($tab_svc) {
-            foreach ($tab_svc as $svc_descr => $svc_id) {
-                $tab = getServiceTemplateCategoryList($svc_id);
+        if ($tabSvc) {
+            foreach (array_keys($tabSvc) as $serviceId) {
+                $tab = getServiceTemplateCategoryList($serviceId);
                 foreach ($tab as $t) {
                     if (isset($authorizedCategories[$t])) {
-                        $tab_services[$svc_descr] = $svc_id;
+                        $tabServices[$serviceId] = 1;
```

Comment:
Why you fix the value "1"?

Reply:
I guess cause isset is better (in performance) than in_array